README.md

```diff
@@ -15,7 +15,7 @@ following:
 
 For Debian and Ubuntu:
 ```
-$ sudo apt-get install git git-svn bc binfmt-support automake autogen autoconf autotools-dev libtool shtool python m4 gcc libtool zlib1g-dev libelf-dev libxml2-dev python-yaml python-pygments pigz pxz
```

Comment:
As of this commit https://github.com/khusika/build-tools-clang/commit/4a6f34e2de381291398a31559fd0f590186d9791, build-essential was removed. So i will reject this commits